README.md

```diff
@@ -46,87 +44,50 @@ or
 yarn global add terminalgpt
 ```
 
-## Run
+## Start chat
 
 ```bash
 tgpt chat
 ```
 
-ps.: If it is your first time running it, it will ask for open AI key , `paste generated key from pre-requisite steps`
+PS: If it is your first time running it, it will ask for open AI key, **paste generated key from pre-requisite steps**.
+
+## Options
 
-## Changing engine and temperature
+### Change engine and temperature
 
 ```bash
-tgpt chat --engine "text-davinci-002" --temperature 0.7
+tgpt chat --engine "gpt-4" --temperature 0.7
 ```
 
-### Changing api key
+Note this library uses [Chat Completions API](https://platform.openai.com/docs/api-reference/chat).
+The `engine` parameter is the same as the `model` parameter in the API. The default value is `gpt-3.5-turbo`.
 
-It you are not satisfy or added a wrong api key , run
+### Use markdown
 
 ```bash
-tgpt delete
+tgpt chat --markdown
 ```
 
-# Using with npx
-
-```bash
-npx terminalgpt
-```
+## Change or delete api key
 
-## Run
+It you are not satisfied or added a wrong api key, run
 
 ```bash
-npx terminalgpt chat
+tgpt delete
 ```
 
-ps.: If it is your first time running it, it will ask for open AI key , `paste generated key from pre-requisite steps`
-
-### Changing engine and temperature
+## Using with npx
 
 ```bash
-npx terminalgpt chat --engine "text-davinci-002" --temperature 0.7
+npx terminalgpt
 ```
 
-### Changing api key
-
-If you are not satisfied or entered a wrong api key, run
-
-```
-npx terminalgpt delete
+```bash
+npx terminalgpt <command>
 ```
 
-## 😎 Contribute your first spec in < 3 minutes
-
-Use the steps below:
-
-<br/>
-
-**Steps**
-
-1. Click [here](https://github.com/jucasoliveira/terminalGPT/fork) to fork this repo.
-
-2. Clone your forked repo and create an example spec
-
-   ```bash
-   # Replace `YOUR_GITHUB_USERNAME` with your own github username
-   git clone https://github.com/YOUR_GITHUB_USERNAME/terminalGPT.git terminalGPT
-   cd terminalGPT
-
-   # Add jucasoliveira/terminalGPT as a remote
-   git remote add upstream https://github.com/jucasoliveira/terminalGPT.git
-
-   # Install packages
-   npm install
-   ```
-
-3. On your terminal, type `npm run chat`. Your terminalGPT will start. 😊
-
-<br>
```

Comment:
Note this entire section is a duplicate and already exists in `CONTRIBUTING.md`.